<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flasher</class>
 <widget class="QDialog" name="Flasher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>553</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>553</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>XSVFPlay FPGA Programming Tool</string>
  </property>
  <widget class="QLineEdit" name="lineFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>451</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonLoad">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonExec">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start FPGA Programming</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textOut">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>511</width>
     <height>211</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>511</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Console Output :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select .xsvf File :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveLog">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>430</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Output Log...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="boardGroup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="spartan6Board">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>SPARTAN6 Board</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Target FPGA Board :</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="virtex6Board1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>VIRTEX6 Board 1.0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
